dnsapi/dns_nsupdate.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
```

Comment:
we need to use `#!/usr/bin/env sh`
we must be `sh` compatible,  in freebsd/openbsd etc,  bash is not available by default.